sla_agent.py

```diff
@@ -184,14 +231,15 @@ def monitor_job(self) -> None:
         try:
             self.logger.info('New monitor job started...')
             skale = spawn_skale_lib(self.skale)
-            try:
-                self.nodes = call_retry.call(skale.monitors.get_checked_array, self.id)
-            except Exception as err:
-                self.logger.exception(f'Failed to get list of monitored nodes. Error: {err}')
-                self.notifier.send(f'Cannot save metrics to database - '
-                                   f'is MySQL container running? {err}', icon=MsgIcon.ERROR)
-                self.logger.info('Monitoring nodes from previous job list')
-
+            # try:
```

Comment:
is this code temporarily commented or should be removed?

Reply:
temporarily commented